### DISCUSSION TITLE
MD in DITA line break after image

### DISCUSSION THREAD

**Discussion by sta1ewar** [dita-ot/dita-ot#4501] [Q&A] [answered]
Hello everyone. I use MD files in DITAMAP and then transform the entire book to HTML. <br> breaks are added after inline graphics. No matter what syntax I use, which dita-ot version and which format (xhtml, html5, iiRDS).
![1_oXygen](https://github.com/user-attachments/assets/33332092-e8bb-48d6-a655-03743096abb5)
![2_html_result](https://github.com/user-attachments/assets/f46f2e8b-629e-43cf-aa7c-5fc5a72bd343)

**Comment by raducoravu** [comment, marked as answer]
Looking at the DITA content generated from the Markdown content by the DITA OT when publishing, it looks like this, it does not have a placement=break attribute:

        <image class="- topic/image " href="..my.jpg"
            xtrf=".."
            xtrc="image:1;12:9" dita-ot:image-width="249" dita-ot:image-height="171"
            dita-ot:horizontal-dpi="150" dita-ot:vertical-dpi="150">
            <alt class="- topic/alt "
                xtrf=".."
                xtrc="alt:1;12:9">washing the car</alt>
        </image>

Then looking at the XSLT template which processes the DITA XML image:
DITA-OT/plugins/org.dita.html5/xsl/topic.xsl

    <xsl:template match="*[contains(@class, ' topic/image ')]" name="topic.image">

it has this condition inside it:

    
    <xsl:if test="not(@placement = 'inline')"><br/></xsl:if>

so it seems to rely on the fact that when regular DITA topics are processed the placement = 'inline' attribute comes as default from the DTDs.

@jelovirt I would see two possible fixes for this:

(1) Change inside the "org.dita.html5/xsl/topic.xsl" the "not(@placement = 'inline')" to "(@placement = 'break')" or
(2) From the DITA to Markdown converter plugin generate in general placement=inline as an attribute on inline image.

**Reply by sta1ewar** [reply]
Thank you so much.
Placement is the keyword.
My fault was to use Display=„inline“.  I use placement=“inline“ now and it works well.

**Comment by raducoravu** [comment]
I also added a DITA OT issue for this: https://github.com/dita-ot/dita-ot/issues/4506
